WebRoot/messages/ZmMsg.properties

```diff
@@ -4626,7 +4626,7 @@ recoveryEmailErrorMessageInvalidCode = Verification of recovery email address ve
 passwordRecoveryTitle= Set up Password Recovery
 passwordRecoveryButtonCancel= Back to sign in
 passwordRecoveryGetAccountDescription = Please enter your username so we may retrieve the recovery email you have on record. \
-<br><br>Don't have a recovery email on record? Please contact your system administrator for help resetting your password.
+<br><br>Don\u2019t have a recovery email on record? Please contact your system administrator for help resetting your password.
```

Comment:
not sure if we need any encoding in english messages

Reply:
I don't know the exact reason, but a single quote `'` has been replaced with `\u2019` on the following PR.
* https://github.com/Zimbra/zm-web-client/pull/134

Reply:
ok got it, I think without the encoding files were not loading properly